.woodpecker/helm.yml

```diff
@@ -0,0 +1,36 @@
+clone:
+  git:
+    image: plugins/git:next
+
+pipeline:
+  release:
+    image: quay.io/helmpack/chart-releaser:v1.2.1
+    secrets:
+      - source: github_token
+        target: CR_TOKEN
+    commands:
+      - sed -i "s/<version>/${DRONE_TAG##v}/g" charts/woodpecker-agent/Chart.yaml
+      - sed -i "s/<version>/${DRONE_TAG##v}/g" charts/woodpecker-server/Chart.yaml
+      - cat charts/woodpecker-agent/Chart.yaml
+      - cat charts/woodpecker-server/Chart.yaml
+      - mkdir -p .cr-index
+      - cr package charts/woodpecker-server
+      - cr package charts/woodpecker-agent
+      - cr upload --owner woodpecker-ci --git-repo woodpecker-ci.github.io --release-name-template "helm-{{ .Name }}-{{ .Version }}"
+      - cr index --owner woodpecker-ci --git-repo woodpecker-ci.github.io --pages-branch main --charts-repo https://woodpecker-ci.github.io --push --release-name-template "helm-{{ .Name }}-{{ .Version }}"
```

Comment:
so if i understand right we also use pages repo for hosting charts?

-> Docusaurus did rm all old files - wouldn't this also be overwritten?

Reply:
That's indeed a problem. Helm releaser would create a new release on the pages repo to save the compressed charts and push an index file containing information about all chart versions. We could try to build the docs and push them by hand to not overwrite the helm index file?

Reply:
It would be nice if Docusaurus has an option to ignore some subvolder/files on deploy ...

(did not looked into it's docu)

Reply:
as per https://github.com/facebook/docusaurus/issues/5567#issuecomment-918425156

we would have to write a own deploy script :/

Reply:
That's a shame, but I can take a look on it.

Reply:
should now be resolved by #333